addOns/webuipoc/src/main/pocs/web-ui/src/App.js

```diff
@@ -0,0 +1,36 @@
+
+import './App.css';
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const App = () => {
+    const [childNode,SetChildNode] = useState(null);
+    const WebClickZAP = async () => {
+        try {
+            const response = await axios.get('http://localhost:1337/JSON/core/view/childNodes/');
```

Comment:
Shouldnt need to specify `http://localhost:1337/` - the API should be on the same host:port
If its not then it will fail anyway..

Reply:
While developing locally, the React development environment runs on `http://localhost:8000` so we need to specify the URL. How can it be ruled out during development?
Yes, I agree once build is deployed, specifying the domain isn't required as it will be on same host:port but thats not the case while developing (even we've to disable CORS temporarily for cross communication)